src/server.h

```diff
@@ -1764,20 +1795,27 @@ struct valkeyServer {
     size_t stat_clients_type_memory[CLIENT_TYPE_COUNT]; /* Mem usage by type */
     size_t stat_cluster_links_memory;                   /* Mem usage by cluster links */
     long long
-        stat_unexpected_error_replies;                 /* Number of unexpected (aof-loading, replica to primary, etc.) error replies */
-    long long stat_total_error_replies;                /* Total number of issued error replies ( command + rejected errors ) */
-    long long stat_dump_payload_sanitizations;         /* Number deep dump payloads integrity validations. */
-    long long stat_io_reads_processed;                 /* Number of read events processed by IO threads */
-    long long stat_io_writes_processed;                /* Number of write events processed by IO threads */
-    long long stat_io_freed_objects;                   /* Number of objects freed by IO threads */
-    long long stat_io_accept_offloaded;                /* Number of offloaded accepts */
-    long long stat_poll_processed_by_io_threads;       /* Total number of poll jobs processed by IO */
-    long long stat_total_reads_processed;              /* Total number of read events processed */
-    long long stat_total_writes_processed;             /* Total number of write events processed */
-    long long stat_client_qbuf_limit_disconnections;   /* Total number of clients reached query buf length limit */
-    long long stat_client_outbuf_limit_disconnections; /* Total number of clients reached output buf length limit */
-    long long stat_total_prefetch_entries;             /* Total number of prefetched dict entries */
-    long long stat_total_prefetch_batches;             /* Total number of prefetched batches */
+        stat_unexpected_error_replies;                  /* Number of unexpected (aof-loading, replica to primary, etc.) error replies */
+    long long stat_total_error_replies;                 /* Total number of issued error replies ( command + rejected errors ) */
```

Comment:
Does this item now fit?
```suggestion
    long long stat_unexpected_error_replies;            /* Number of unexpected (aof-loading, replica to primary, etc.) error replies */
    long long stat_total_error_replies;                 /* Total number of issued error replies ( command + rejected errors ) */
```